src/main/java/io/elastic/jdbc/ColumnNamesProviderOld.java

```diff
@@ -0,0 +1,116 @@
+package io.elastic.jdbc;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import io.elastic.api.DynamicMetadataProvider;
+import io.elastic.api.SelectModelProvider;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.*;
```

Comment:
Please resolve all the bulk dependencies import.  '.*'-style import should not be used